src/form-elements/FormElementCalculation.tsx

```diff
@@ -241,32 +219,22 @@ function FormElementCalculation({
   // MODEL LISTENER
   React.useEffect(() => {
     if (!calculation) return
-    const submission = getFormSubmissionModel(false).submission
-    const newValue = calculation.eval(submission)
-    if (value === undefined) {
-      if (Object.is(NaN, newValue)) return
```

Comment:
I have a memory that there was a  reason for using `Object.is` here..

Reply:
`isNaN(newValue)` should be better suited here as `newValue` is not typed so it could be falsey.